classes/CartRule.php

```diff
@@ -1313,7 +1319,13 @@ protected function getCartRuleCombinations($offset = null, $limit = null, $searc
      * @return array|bool Array with DB rows of requested type
      * @throws PrestaShopDatabaseException
      */
-    public function getAssociatedRestrictions($type, $active_only, $i18n, $offset = null, $limit = null, $search_cart_rule_name = '')
+    public function getAssociatedRestrictions(
+        $type,
+        $active_only,
+        $i18n, $offset = null,
```

Comment:
If you wanted to write param perline, maybe you would like to separate `$i18n` from `$offset` ?